tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -425,22 +424,70 @@ def test_pantsd_killed_run(self):
       )
 
       # Wait for the python run to be running.
-      time.sleep(15)
+      time.sleep(5)
       checker.assert_started()
 
       # Locate the single "parent" pantsd-runner process, and kill it.
       pantsd_runner_processes = [p for p in checker.runner_process_context.current_processes()
                                  if p.ppid() == 1]
       self.assertEquals(1, len(pantsd_runner_processes))
-      pantsd_runner_processes[0].terminate()
+      parent_runner_process = pantsd_runner_processes[0]
+      parent_runner_pid = parent_runner_process.pid
+      # Send SIGTERM
+      parent_runner_process.terminate()
       waiter_run = waiter_handle.join()
 
       # Ensure that we saw the failure in the client's stdout, and that we got a remote exception.
       self.assert_failure(waiter_run)
-      self.assertIn('abruptly lost active connection to pantsd runner', waiter_run.stderr_data)
+      self.assertRegexpMatches(waiter_run.stderr_data, """\
+Signal {signum} was raised\\. Exiting with failure\\.
+\\(backtrace omitted\\)
+""".format(pid=parent_runner_pid, signum=signal.SIGTERM))
```

Comment:
The `pid` isn't used here, but that's probably fine: telling someone the dead pid is less necessary than rendering the error.

Reply:
This is an artifact of a previous implementation which logged the pid with a timestamp to stderr, which is only supposed to occur when writing to a log file, which is why this error stayed hidden. I think it can be removed.

Reply:
This has been removed, with an note about why we only need to check the stderr.